src/index.js

```diff
@@ -11,8 +11,12 @@ const rootElement = document.getElementById('root');
 ReactDOM.render(
   <Router>
     <Switch>
-      <Route exact path="/" render={() => <App demo={false} />}></Route>
-      <Route path="/demo" render={() => <App demo={true} />}></Route>
+      <Route exact path="/" render={() => <App demoData={null} />}></Route>
```

Comment:
Looks like `demoData` was removed from the props interface so it should be deleted here
```suggestion
      <Route exact path="/" render={() => <App  />}></Route>
```

Reply:
updated to demoId={null}

Reply:
`demoId` is optional so it can be removed entirely and just be `<App />`